examples/features/grid-filter.js

```diff
@@ -36,6 +37,7 @@ Ext.application({
             format: new ol.format.GeoJSON({
                 featureProjection: 'EPSG:3857'
             }),
+            features: new ol.Collection(),
```

Comment:
```suggestion
```

This line should not be necessary. The layer does not handle a `features` parameter as far as I know. The important part is the `features` parameter in the source.